app/views/group_orders/_form.html.haml

```diff
@@ -156,13 +160,19 @@
           %td.currency
             %span#total_price= number_to_currency(@group_order.price)
         %tr
-          %td= heading_helper(Ordergroup, :available_funds) + ':'
-          %td.currency= number_to_currency(@ordering_data[:available_funds])
+          - if FoodsoftConfig[:charge_members_manually]
+            - old_balance = @ordering_data[:account_balance]
+            %td= heading_helper(Ordergroup, :account_balance) + ':'
+            %td.currency= number_to_currency(@ordering_data[:account_balance])
+          - else
+            - old_balance = @ordering_data[:available_funds]
+            %td= heading_helper(Ordergroup, :available_funds) + ':'
+            %td.currency= number_to_currency(@ordering_data[:available_funds])
         %tr
           %td= t('.new_funds') + ':'
           %td.currency
             %strong
-              %span#new_balance= number_to_currency(@ordering_data[:available_funds] - @group_order.price)
+              %span#new_balance= number_to_currency(old_balance - @group_order.price)
```

Comment:
Would this work when there are multiple open orders? At a glance it looks like only the order currently viewed is substracted from the balance for manual charging.

Reply:
The `charge_members_manually` mode works as intended: The calculation uses only the current account balance, so no other orders are involved. For the "normal" mode, it works like without this change.

Reply:
I can see this work well with `charge_members_manually` when there is one open order. In other cases, only the current group order total is substracted from the account balance, while there may be other open orders. As a member, I would find it confusing. If just the old balance would be shown, which doesn't change when the group order changes, that may be clearer.

There have been many situations where ordering users didn't really understand what Foodsoft was doing and why, that's why I'm sensitive to things that may be confusing, especially regarding numbers and amounts.

But then I don't have a clear overview of the whole manual ordering flow, so I leave this to your judgement.

Reply:
i agree with your point. IMHO the best solution would be to hide it completely, but that would require a bigger UI change (since members shouldn't be allowed to buy unlimited stuff). The relevant change is, that there is no need to put money to the ordergroup and it's possible to have unlimited `finished` an not `closed` orders (let's just ignore the fact, that having a lot of `finished` orders is not a good idea).